### DISCUSSION TITLE
How to include a # comment inside a """ docstring block in Python

### DISCUSSION THREAD

**Discussion by rajk-cerebras** [executablebooks/meta#348] [Q&A]
I am not sure if this is a Sphinx or Sphinx Book Theme-related or Python docstring-related question. Does anyone know how to insert a `# comment line` within a `"""` docstring block in Python in a way the Sphinx Book Theme honors it? For example, I need to exclude two or three arguments from rendering in HTML so I tried using it, unsuccessfully, like the following:

```
Args:
        somearg (int): descriptiontext.
        someotherarg (callable): someotherdescription
        # IGNORE-BEGIN
        anotherarg (bool): Specifies ... whatever.
        # IGNORE-END
        somearg3 (int): descriptiontext.
        somearg4 (bool): descriptiontext
```
I want the `anotherarg` not be included in the generated HTML. I also tried the following without success:

```
Args:
        somearg (int): descriptiontext.
        someotherarg (callable): someotherdescription
.. IGNORE-BEGIN
        anotherarg (bool): Specifies ... whatever.
        # IGNORE-END

        somearg3 (int): descriptiontext.
        somearg4 (bool): descriptiontext
```
Anyone has ideas? Thanks!

**Comment by choldgraf** [comment]
I’m not quite sure what you’re trying to do. Do you want the comments to show up or not? This feels like it is unrelated to the book theme and is more like a python question since Sphinx doesn’t treat strings any differently than other applications, I think
